bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v3/ImageImpl.java

```diff
@@ -15,36 +15,44 @@
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 package com.adobe.cq.wcm.core.components.internal.models.v3;
 
-import java.io.UnsupportedEncodingException;
-import java.net.URLDecoder;
-import java.nio.charset.StandardCharsets;
-
-import javax.annotation.PostConstruct;
-
+import com.adobe.cq.export.json.ComponentExporter;
```

Comment:
Please reorganize the imports according to the AEM code conventions:
https://wiki.corp.adobe.com/display/tsg/Code+Conventions+For+Java